workers/unity/Packages/com.improbable.gdk.guns/Behaviours/ClientShooting.cs

```diff
@@ -87,7 +87,7 @@ public void FireShot(float range, Ray ray)
                 EntityId = entityId,
                 HitSomething = hitSomething,
                 HitLocation = (hitLocation - spatial.Worker.Origin).ToIntAbsolute(),
-                HitOrigin = (ray.origin - spatial.Worker.Origin).ToIntAbsolute()
+                HitOrigin = (ray.origin - spatial.Worker.Origin).ToIntAbsolute(),
```

Comment:
why did it add the `,`

Reply:
ah, cause I had modified the schema in between and then reverted it. in general I prefer to have a comma there, so I think it's fine to leave it as is?

Reply:
ah fair enough, just curious